### PR TITLE
<RadioGroup/>, <MessageModelLayout/>, <CustomModalLayout/>, <BaseModalLayout/> - support using dir="rtl" instead of className="rtl"

### PR DESCRIPTION
**Pull request by sivanharel**
…lLayout/> - support dir="rtl" instead of className .rtl

### 🔦 Summary

`<RadioGroup/>`, `<MessageModelLayout/>`, `<CustomModalLayout/>`, `<BaseModalLayout/>` - support using dir="rtl" instead of className="rtl"

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/RadioGroup/RadioGroup.st.css [hunk 1]
```diff
@@ -138,33 +138,33 @@
   align-items: center;
 }
 
-:global(.rtl) .optionWrapper {
+:global([dir='rtl']) .optionWrapper {
   direction: rtl;
 }
```
Comment (sivanharel) [inline src/RadioGroup/RadioGroup.st.css:143]: It seems like there is no need for it.
Am I missing something?

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5941/

File: src/RadioGroup/RadioGroup.st.css (hunk 1 above)
Comment (shlomitc) [reply src/RadioGroup/RadioGroup.st.css:143]: does seems redundant

Comment (sivanharel) [reply src/RadioGroup/RadioGroup.st.css]: great so I am removing it.

**Review by shlomitc — APPROVED** [review]

**Pull request merged**